features/step_definitions/datailTourPageSteps.js

```diff
@@ -0,0 +1,16 @@
+import DetailTourPage from '../utils/pageobjects/detail.tour.page';
+
+const { When, Then } = require('cucumber');
+
+
+Then(/^On detail tour page amount of adults should be (.*)$/, async (adults) => {
+  await expect(browser).toHaveUrlContaining(adults);
+});
+
+When(/^I book the tour$/, async () => {
+  await DetailTourPage.choseMaximumAmountOfDays();
+});
+
+When(/^Fill form with (.*), (.*), (.*), (.*)$/, async (name, email, phone, address) => {
```

Comment:
I'd make test data to be stored in a separate file. And here path the name of the data (e.g.`user1`)

export const TestData = {
  user1: {name, email, ...},
  user2: {name, email, ...}
}